core/lib/multivm/src/versions/vm_latest/tracers/result_tracer.rs

```diff
@@ -101,6 +168,29 @@ impl<S, H: HistoryMode> DynTracer<S, SimpleMemory<H>> for ResultTracer<S> {
                 });
             }
         }
+
+        if state.vm_local_state.callstack.current.this_address == BOOTLOADER_ADDRESS {
+            let opcode_variant = data.opcode.variant;
+            if let Opcode::FarCall(_) = opcode_variant.opcode {
+                self.far_call_tracker.far_call_observed(&state);
+            }
+        }
+    }
+
+    fn after_execution(
+        &mut self,
+        state: VmLocalStateData<'_>,
+        data: zk_evm_1_5_0::tracing::AfterExecutionData,
+        _memory: &SimpleMemory<H>,
+        _storage: StoragePtr<S>,
+    ) {
+        let opcode_variant = data.opcode.variant;
+
+        if state.vm_local_state.callstack.current.this_address == BOOTLOADER_ADDRESS {
```

Comment:
As I remember we already made a pop at this place and this address is not a bootloader